wrappers/python/examples/wnt/README.md

```diff
@@ -0,0 +1,64 @@
+# WNT API Examples
+
+This folder contains the WNT API Examples.
+
+## Installation
+
+The examples require that the wirepas_messaging packet is installed. Please see the README.md
+file in /wrappers/python folder for installation instructions.
+
+## WNT backend settings
+
+For running the examples, WNT backend connection parameters needs be given by creating settings.yml
+file to the same folder from where the examples are run. For example settings file please see
+settings.yml in the root folder.
+
+Please note that "operator_username" and "operator_password" properties are not needed for running the examples.
+
+## Examples
+
+### applicationconfiguration.py
+Application configuration and diagnostics interval setting *
+
+### authentication.py
+Authentication and users manipulation
+
+### building.py
+Building manipulation
```

Comment:
What do you mean by building manipulation? Could you describe what is does?

Reply:
Done